package.json

```diff
@@ -124,10 +124,10 @@
 		"vitest": "^2.0.3",
 		"webpack": "^5.93.0",
 		"webpack-cli": "^5.1.4",
-		"xo": "^0.59.0"
+		"xo": "^0.59.1"
 	},
 	"engines": {
-		"node": "~21",
+		"node": ">= 20",
```

Comment:
The tilde could be dropped because the npm issue was fixed:

- https://github.com/npm/cli/issues/7657